package.json

```diff
@@ -63,8 +64,8 @@
     "input-otp": "^1.4.2",
     "lucide-react": "^0.509.0",
```

Comment:
So, is this still needed then? 🤔

Reply:
@RavenHursT I was playing it safe with backwards compatibility, but yolo I'll remove. Might come up if we import other components.